widgets/menus.pas

```diff
@@ -0,0 +1,93 @@
+unit Menus;
```

Comment:
Please add a copyright header as in the ```Grids``` unit (with Modified LGPL)

Reply:
I will do it.